From b8bc4e83bf447b6330ece54e94a1684d12ecdd0b Mon Sep 17 00:00:00 2001
From: Marcin Niestroj <m.niestroj@grinn-global.com>
Date: Fri, 23 Jun 2017 12:00:30 +0200
Subject: [PATCH] common: Include sys/types.h header
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Build with musl library fails with following error:

In file included from src/crypto/sha1.c:21:0:
./src/digest.h:95:10: error: unknown type name ‘ulong’
          ulong start, ulong size);
...

Fix that by including sys/types.h header in common.h, so ulong type
is defined with musl library.

Signed-off-by: Marcin Niestroj <m.niestroj@grinn-global.com>
---
 src/dt/common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/dt/common.h b/src/dt/common.h
index 992e28f..1425c53 100644
--- a/src/dt/common.h
+++ b/src/dt/common.h
@@ -13,6 +13,7 @@
 
 #include <sys/ioctl.h>
 #include <sys/stat.h>
+#include <sys/types.h>
 
 #include <mtd/mtd-abi.h>
 
-- 
2.13.1

